controller.js

```diff
@@ -1,7 +1,89 @@
-var Calculator = require( './Calculator' )
-var clickHandler = document.getElementById( 'calculator-keypad' )
+var Calc = new Calculator() //eslint-disable-line
+var clickElement = document.querySelector( '.calculator-keypad' )
 
-clickHandler.addEventListener( '.click', function( event ) {
-  var value = event.target
-  console.log( event.target )
-} )
+clickElement.addEventListener( 'click', calculatorHandler.bind( this ) )
+document.addEventListener( 'keypress', calculatorHandler.bind( this ) )
+
+function calculatorHandler ( event ) {
+  var displayElement = document.querySelector( '.calculator-display' )
+  var eventKey
+
+  if ( event.key ) {
+    eventKey = parsedKey( event.key )
+  }
+
+  var value = ( !event.key ) ? event.target.value : eventKey
+  var calculatorButton = document.querySelector( `[value='${eventKey}']` )
```

Comment:
You can get this element from `event.target` although you might have to walk up the parents to get to the `<button>` element